templates/todo/tag_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
```

Comment:
Loading crispy_forms_filters to use crispy forms template tags is appropriate if you are using Django Crispy Forms for better form rendering.